params.yml

```
@@ -52,25 +52,25 @@ args:
 
   # Docker images and repositories used to spin up services.
   # zkevm_prover_image: hermeznetwork/zkevm-prover:v8.0.0-RC5-fork.12
-  zkevm_prover_image: hermeznetwork/zkevm-prover:v6.0.3-RC20
+  zkevm_prover_image: hermeznetwork/zkevm-prover:v8.0.0-RC5-fork.12
   # zkevm_prover_image: hermeznetwork/zkevm-prover:v4.0.19
 
   zkevm_node_image: hermeznetwork/zkevm-node:v0.7.0
   cdk_validium_node_image: 0xpolygon/cdk-validium-node:0.7.0-cdk
-  cdk_node_image: ghcr.io/0xpolygon/cdk:0.0.9
+  cdk_node_image: arnaubennassar/cdk:use-l1info-7
 
-  zkevm_da_image: 0xpolygon/cdk-data-availability:0.0.7
+  zkevm_da_image: arnaubennassar/cdk-data-availability:fix-banana
 
-  zkevm_contracts_image: leovct/zkevm-contracts # the tag is automatically replaced by the value of /zkevm_rollup_fork_id/
+  zkevm_contracts_image: arnaubennassar/zkevm-contracts   # the tag is automatically replaced by the value of /zkevm_rollup_fork_id/
```

I just build a [new image](https://hub.docker.com/layers/leovct/zkevm-contracts/fork12/images/sha256-8c6028410e6089e99d4696a59032d553bf8a8d9e228dca9a07289c0f6df0674b?context=repo) to support `fork12`.

```suggestion
  zkevm_contracts_image: leovct/zkevm-contracts   # the tag is automatically replaced by the value of /zkevm_rollup_fork_id/
```

---

Your image only supports `fork12` unfortunately... We'd like to be able to support all kind of tests and being able to test older forks is still needed...